README.md

```diff
@@ -373,15 +373,22 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The proxy submit commands add two parameters to the normal submission command:
```

Comment:
This section is no longer specific to proxy/submit, so you should talk about these options in a more general sense, e.g.,
"The family of proxy commands use the following two new concepts: ...".

You are usually good about this, but I just want to reinforce that when I provide examples they don't have to be followed exactly.
They can, but exactly how you want to write it is up to you.